src/service/sandbox-svc.ts

```diff
@@ -64,9 +61,11 @@ function sendEdgeRequest(pth: string, method: string, body, headers, filePath?:
         } else {
           try {
             const errorObj = JSON.parse(body);
+            errorObj.path = path
+            errorObj.method = method
             reject(cliUtils.toJsonPretty(errorObj));
           } catch (ex) {
-            console.error(`got error code: ${response.statusCode} calling ${method} ${path}\n${body}`);
+            console.error(`ERROR: got error code: ${response.statusCode} calling ${method} ${path}\n${body}`);
```

Comment:
I've just added it to  be consistent with other places - errors should be easily visible in execution logs.

Reply:
Yes, but I did't question adding that. Just that we have a lot of `ERROR: ` strings in here, instead of doing that we should have some common function that prepends errors and does `console.error`.

Something like `logError('some error')` should execute `console.error('ERROR: some error');`
`logInfo('some information'` should execute `console.info('some information')`.

Reply:
I created logError as you suggested